go/src/meguca/websockets/feeds/feeds.go

```diff
@@ -6,11 +6,36 @@ package feeds
 import (
 	"errors"
 	"sync"
+	"time"
 
 	"meguca/auth"
 	"meguca/common"
 )
 
+func init() {
```

Comment:
Maybe we should move this over to clients.go so we don't have to do a mutex lock on feeds?
Not sure how that will affect this.